src/common-utils/devcontainer-feature.json

```diff
@@ -10,6 +10,11 @@
             "default": true,
```

Comment:
Can we bump the minor version in here? 

https://github.com/devcontainers/features/blob/1c60fd71daa84c5fb81b4daf499974dbffb9e029/src/common-utils/devcontainer-feature.json#L3